docs/src/plots/TerminalVelocity.jl

```diff
@@ -98,6 +98,25 @@ function rain_terminal_velocity_individual_SB(
     return v = (ρ0 / ρ)^(1 / 2) * (aR - bR * exp(-cR * D_r))
 end
 
+"""
+    rstokes_regime_terminal_velocity(param_set, ρ, r)
+
+ - `param_set` - set with free parameters
+ - `ρ` - air density
+ - `r` - radius of the cloud droplets
+
+Returns the fall velocity of a cloud droplet in the Stokes regime (Re < 1)
+"""
+function stokes_regime_terminal_velocity(
```

Comment:
I think there is a use case to add it to `src`. @haakon-e needs particle-level formulation of sedimentation velocity, because he is doing numerical integrals in P3 scheme to compute the collision rate. Maybe we could add it to `Common` next to `Chen` particle velocities? 

Along the same lines, do you think SB scheme would be ok to use with Chen 2022 for sedimentation velocity for rain? That one doesn't have the issue of negative numbers close to zero, so it's easier to plug in into the P3 scheme integrals. 

The relevant code is here: https://github.com/CliMA/CloudMicrophysics.jl/blob/7d30216674cda6668c959f8eba40bad1220370d7/src/P3_processes.jl#L133